allthedoll-seattle.md

```diff
@@ -0,0 +1,4 @@
+## Seattle Recommendationz
+
+- Item 1
```

Comment:
@allthedoll Item 1 is a terrible recommendation. Can you please be more descriptive? 👎